app/com/linkedin/drelephant/analysis/AnalyticJobGeneratorHadoop2.java

```diff
@@ -46,10 +46,14 @@
   private static final String IS_RM_HA_ENABLED = "yarn.resourcemanager.ha.enabled";
   private static final String RESOURCE_MANAGER_IDS = "yarn.resourcemanager.ha.rm-ids";
   private static final String RM_NODE_STATE_URL = "http://%s/ws/v1/cluster/info";
+  private static final String FETCH_ONLY_RECENT_RM_APPS = "drelephant.analysis.only.fetchRecentApps";
```

Comment:
Addressed your comment. Please take a look.
I agree that we should organize config files more clearly. Maybe, there should be just one config file, but I think we can address that later.